### ISSUE TITLE
Can't download map using Navit menu on Android

### ISSUE THREAD

**Issue by marcin-github** [navit-gps/navit#1132]
Navit-0.5.5 from GPlay.
Since many weeks I can't download maps using builitn option. Download starts and after a couple of seconds transfer is stalled. Navit retries to download but it always finish after small amount of data.

**Comment by mvglasow** [comment]
Yes, we are currently having issues with the map server. Unfortunately, the maintainer of the server seems to have left the project. @jkoan is working on a solution.

Currently your options are:
* look https://github.com/jkoan/gh-actions-mapserver (under Releases), maybe one of the map files there suits your needs
* A map of Europe is available as a bit torrent at magnet:?xt=urn:btih:29e637e2dd8e376c85e1abe3445606e7277db38f&dn=europe-210503.bin (19.49 GB in size); I am currently seeding

**Comment by marcin-github** [comment]
Thank you for information

**Comment by hoehnp** [comment]
As I feel a little
